src/components/User.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+export default function User(props) {
+  const {name, email, address} = props;
```

Comment:
should be the same style everywhere `{name, email, address}` OR `{ name, email, address }`